guard/tests/utils.rs

```diff
@@ -34,7 +35,7 @@ pub fn get_full_path_for_resource_file(path: &str) -> String {
     return resource.display().to_string();
 }
 
-pub fn cfn_guard_test_command(args: Vec<&str>) -> i32 {
+pub fn cfn_guard_test_command(args: Vec<&str>, command: Box<dyn Command>) -> i32 {
```

Comment:
Does this need to be a `Box`? Can this be a `Command` instead? Also, let's change the order of arguments, Command first followed by args.

Reply:
No it didnt, just figured since it needed to be a Box to be pushed to the vec it made sense. I updated the function signature, to take a generic T: Command, and just wrap it in the box when we push to the vec.